src/traces/parcoords/utils.js

```diff
@@ -0,0 +1,137 @@
+/**
```

Comment:
naming modules like these `helpers.js` would be more inline with the rest of the repo.